codice/ddf
# Upgrade to Solr 8.6.1

#### What does this PR do?
Upgrades Solr and converts the SolrCloud docker compose files to a maven project.

#### Who is reviewing it? 

@lamhuy 
@derekwilhelm 

#### Select relevant component teams: 

@codice/solr 

#### Ask 2 committers to review/merge the PR and tag them here.

@rzwiefel
@lambeaux

#### How should this be tested?

Start SolrCloud using `distribution\docker\solrcloud`. Verify data can be ingested and searched.

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: n/a

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
